static/app/views/organizationGroupDetails/groupEventAttachments/screenshotCard.tsx

```diff
@@ -62,7 +81,19 @@ export function ScreenshotCard({
     <Card>
       <CardHeader>
         <CardContent>
-          <Title to={`${baseEventsPath}${eventId}/`}>{eventId}</Title>
+          <Title
+            onClick={() =>
+              trackAdvancedAnalyticsEvent(
+                'issue_details.attachment_tab.screenshot_card_clicked',
+                {
+                  organization,
+                }
+              )
+            }
```

Comment:
This links out to the event so `screenshot_card_clicked` doesn't seem like it fits here, should this be renamed to make it more clear that users are clicking the event ID to navigate to the event?

Reply:
Hmm I intentionally called this card clicked since its not necessarily true that clicking the card will always lead to event id

Reply:
But this is on the `Title` component that is actually a styled Link, right? Would it make more sense to trigger this event from a higher component like `Card`? e.g. footer clicks wouldn't be captured with this